docker/environment.yml

```diff
@@ -22,11 +36,11 @@ dependencies:
   - descartes
   # Pin rasterio for ravenpy, remove on next build.
   # See https://github.com/CSHS-CWRA/RavenPy/commit/eae66e9afc30e2381e9119644a0695d1d248c739
-  - rasterio <= 1.2.6
-  - gdal  # for osgeo
-  - geopandas
-  - pandas
-  - rioxarray
+    # - rasterio <= 1.2.6  # from ravenpy
+    # - gdal  # for osgeo, from ravenpy
```

Comment:
```suggestion
```

Reply:
Do you actually mean gdal is not a dependency from ravenpy?  But it still is https://github.com/conda-forge/ravenpy-feedstock/blob/ae682f7bb3fc586737733d923111d3d2107f5ae0/recipe/meta.yaml#L34

Reply:
No, no, GDAL is all-knowing and ever-present. I figured there's no major reason for specifying it to be installed since it comes bundled with so many other packages.

Reply:
Oh I see what you mean.  But according to this commit https://github.com/Ouranosinc/PAVICS-e2e-workflow-tests/commit/d2948dcf1cfec0dd2aa07ee784907420bb5586ee, some notebooks was actually importing it directly and it was not yet in the Jupyter env.

I'd keep it here for historical reason.